source/storing-data/archive-versions.rst

```diff
@@ -0,0 +1,240 @@
+Archive Versions
+================
+.. contents::
+   :local:
+
+The structure of QIIME 2 :term:`archives<Archive>` has evolved as QIIME 2 has been developed.
+This page describes each historical version of the QIIME 2 Archive format,
+and may be useful to interface developers whose code depends on guarantees made by that format
+(`source code <https://github.com/qiime2/qiime2/blob/master/qiime2/core/archive/>`_).
+
+For general information about the structure of current QIIME 2 Archives, see :doc:`archive`.
+For a detailed description of the part of an Archive which holds Provenance data, see :doc:`provenance`.
+
+Version-agnostic format "guarantees"
+------------------------------------
+
+Though there is significant variability in the format of QIIME 2 Archives across archive versions,
+all archive versions share some common traits.
+
+These shared characteristics, defined in the ``_Archive`` class in
+`qiime2/core/archive/archiver.py <https://github.com/qiime2/qiime2/blob/master/qiime2/core/archive/archiver.py>`_,
+must be consistent across all formats over time, 
+as they allow archive versions to be checked,
+and archives with different formats to be dispatched to the appropriate version-specific tools.
+
+All QIIME 2 Archives have:
+
+- a directory named with the Archive UUID, directly under the archive root at ``/<UUID>/``
+- a file ``/<UUID>/VERSION`` within that directory, formatted as shown below
+
+The Archive file system must look like this:
+
+.. figure:: ../img/archive_versions/format_agnostic_archive_structure.svg
+   :alt: Box and Arrow diagram of the guaranteed components of an archive, as described above.
+
+The VERSION file must look like this:
+
+.. code-block::
+
+    QIIME 2
+    archive: <archive version>
+    framework: <framework version>
+
+.. note::
+   This file is intentionally not in YAML, INI,
+   or any other common data serialization or configuration format.
+   This is to discourage the situation where important archive files are reformatted
+   from YAML to another format and VERSION is updated "for consistency",
+   breaking backwards compatibility.
+
+Version 0
+---------
+
+The original QIIME 2 Archive format, there aren't many V0 Archives "in the wild".
+V0 Archives were produced by Alpha versions of the QIIME 2 framework,
+and were superseded in framework version 2.0.6 on 10/24/16.
+
+- :term:`Result` data files are written in the directory ``/<UUID>/data``
```

Comment:
:term:`Resultant` data files are written in the directory ``/<UUID>/data``

Small change here - I think this is more grammatically correct